src/main/java/org/mule/extension/db/internal/domain/connection/oracle/OracleDbConnection.java

```diff
@@ -72,8 +73,6 @@
   private static final int PROCEDURE_NAME = 3;
   private static final int PARAM_NAME_COLUMN_INDEX = 4;
 
-  private Method createArrayMethod;
-
   Map<String, Map<Integer, ResolvedDbType>> resolvedDbTypesCache;
```

Comment:
Should this be final? (resolvedDbTypesCache)

Reply:
Yes, totally, and private as well.